stacker_blueprints/firehose/base.py

```diff
@@ -214,10 +211,9 @@ class Firehose(Blueprint):
                            "write to the firehose stream.",
             "default": [],
         },
-        "BucketName": {
+        "ExistingBucketName": {
             "type": str,
-            "description": "Name for the S3 Bucket",
-            "default": "",
+            "description": "Name of the existing bucket"
         },
         "EncryptS3Bucket": {
```

Comment:
We talkeda bout this, but I think this variable name is incorrect and should better reflect what it actually does.